### DISCUSSION TITLE
Secondary disk (imagefs)

### DISCUSSION THREAD

**Discussion by cyrus-mc** [bottlerocket-os/bottlerocket#2136] [Q&A] [answered]
Is it possible to add a secondary disk to be used for `imagefs` (/var/lib/containerd)?

**Comment by etungsten** [comment, marked as answer]
Hi @cyrus-mc, it's definitely possible. Here is an example of how to use ephemeral disks as the image store for containerd with the use of a [bootstrap-container](https://github.com/bottlerocket-os/bottlerocket#bootstrap-containers-settings): https://github.com/bottlerocket-os/bottlerocket/discussions/1991. Let us know if that example works for you.

**Comment by cyrus-mc** [comment]
@etungsten That makes sense. Thanks.

One quick follow-up question if you don't mind. What is the disk layout of Bottlerocket? I am seeing that if I create a launch configuration with two disks (xvda - 2g and sdf with 100) the actual instance launches with three disks

* xvda - 2g
* xvdb - 20g
* sdf - 100g

**Reply by webern** [reply]
I believe in your example you're seeing
- provided by the AMI definition
  - xvda: root filesystem (EBS)
  - xvdb: data partition (EBS): used for bottlerocket settings, updates, kublet pki files and other non-ephemeral data. This is also where your container images are stored unless you mount storage as described in #1991
- added by your launch template
  - sdf: your additional storage
